src/Suave/Suave.fsproj

```diff
@@ -196,4 +196,89 @@
       </ItemGroup>
     </When>
   </Choose>
+  <Choose>
```

Comment:
I will revert this if it gets merged, it's massively breaking and a pain in the ass to work with; like I've said three times so far, if you want to use Newtonsoft.Json then either ILMerge it or make it a stand-alone lib.